### DISCUSSION TITLE
https://penguins--eggs-net.translate.goog/blog/un-punto-morto?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true

### DISCUSSION THREAD

**Discussion by giscus** [pieroproietti/penguins-blog#11] [General]
# https://penguins--eggs-net.translate.goog/blog/un-punto-morto?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true

Eggs è nato più o meno nel corso del 2016 sulle ceneri di software precedenti.

https://penguins--eggs-net.translate.goog/blog/un-punto-morto?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true

**Comment by doodstecsonhere** [comment]
Supporting Fedora would be wonderful! Arch user here. :) Still figuring out how to configure calamares with btrfs.

**Reply by pieroproietti** [reply]
Hi @doodstecsonhere 
I put on [testing](https://sourceforge.net/projects/penguins-eggs/files/DEBS/testing/) a new version for Arch and Debian. 

Here I changed the way the configuration of calamares work, it's not more necessary to use `--theme spiral`, automatically eggs detect from the packages in the system that filesystem is applicable. 

The available filesystems are: ext4, btrfs, xfs and f2fs.
